backend/src/main/java/com/festago/mock/MockDataService.java

```diff
@@ -0,0 +1,168 @@
+import com.festago.festival.domain.Festival;
+import com.festago.festival.dto.command.FestivalCreateCommand;
+import com.festago.festival.repository.FestivalRepository;
+import com.festago.school.application.SchoolCommandService;
+import com.festago.school.domain.School;
+import com.festago.school.domain.SchoolRegion;
+import com.festago.school.dto.SchoolCreateCommand;
+import com.festago.school.repository.SchoolRepository;
+import com.festago.stage.application.command.StageCommandFacadeService;
+import com.festago.stage.dto.command.StageCreateCommand;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Queue;
+import java.util.concurrent.atomic.AtomicLong;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Profile({"local"})
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class MockDataService {
+
+    private static final AtomicLong FESTIVAL_SEQUENCE = new AtomicLong();
+    private static final long STAGE_START_HOUR = 19L;
+    private static final int STAGE_ARTIST_COUNT = 3;
+
+    private final FestivalDateGenerator festivalDateGenerator;
+    private final SchoolRepository schoolRepository;
+    private final ArtistRepository artistRepository;
+    private final FestivalRepository festivalRepository;
+    private final FestivalCommandFacadeService festivalCommandFacadeService;
+    private final StageCommandFacadeService stageCommandFacadeService;
+    private final ArtistCommandService artistCommandService;
+    private final SchoolCommandService schoolCommandService;
+
+
+    public void initialize() {
+        if (alreadyInitialized()) {
+            return;
+        }
+        initializeData();
+    }
+
+    private boolean alreadyInitialized() {
+        return !schoolRepository.findAll().isEmpty();
+    }
+
+    private void initializeData() {
+        initializeSchool();
+        initializeArtist();
+    }
+
+    private void initializeSchool() {
+        for (SchoolRegion schoolRegion : SchoolRegion.values()) {
+            if (SchoolRegion.ANY.equals(schoolRegion)) {
+                continue;
+            }
+            makeRegionSchools(schoolRegion);
+        }
+    }
+
+    private void makeRegionSchools(SchoolRegion schoolRegion) {
+        for (int i = 0; i < 3; i++) {
+            int schoolNumber = i + 1;
+            String schoolName = schoolRegion.name() + schoolNumber;
+            schoolCommandService.createSchool(new SchoolCreateCommand(
+                    schoolName,
+                    schoolName + ".com",
+                    schoolRegion,
+                    null,
+                    null
+                )
+            );
+        }
+    }
```

Comment:
이슈 내용에는 나와있는데, 나중에 코드만 보면 어떤 일을 하는지 알기 힘들 것 같네요.
주석을 남겨주는게 어떤가요?
```suggestion
    /**
    * 설명
    */
    private void makeRegionSchools(SchoolRegion schoolRegion) {
        for (int schoolNumber = 1; schoolNumber <= 3; i++) {
            String schoolName = schoolRegion.name() + schoolNumber;
            schoolCommandService.createSchool(new SchoolCreateCommand(
                    schoolName,
                    schoolName + ".com",
                    schoolRegion,
                    null,
                    null
                )
            );
        }
    }
```